index.html

```diff
@@ -31,11 +31,11 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
             <div class="lets-connect">LET'S CONNECT</div>
             <div class="lets-connect-icons">
                <ul>
-                  <li><a href="https://twitter.com/Abiy62463489"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
-                  <li><a href="https://www.linkedin.com/in/abiy-hussen-aman-194587183/"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
-                  <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
-                  <li><a href="https://github.com/abiy006"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
-                  <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+                  <li><a href="https://twitter.com/Abiy62463489"><img src="Images\Vector1.svg" alt="Social Media Icon" width="19.99px" height="19.96px"></a></li>
+                  <li><a href="https://www.linkedin.com/in/abiy-hussen-aman-194587183/"><img src="Images\Vector2.svg" alt="Social Media Icon" width="20px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector3.svg" alt="Social Media Icon" width="20px" height="16px"></a></li>
+                  <li><a href="https://github.com/abiy006"><img src="Images\Vector4.svg" alt="Social Media Icon" width="17.99px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector5.svg" alt="Social Media Icon" width="14.17px" height="20px"></a></li>
```

Comment:
Kindly ensure that the `alternative text` provides adequate information for the missing images (per the Image text alternatives section above). let's be more specific e.g using `Twitter Icon` instead of `Social Media Icon`

Appropriate alternative text is useful for interacting with or understanding the page. i.e

```html
<li><a href=""><img src="Images\Vector1.svg" alt="Twitter Icon" width="" height=""></a></li>
<li><a href="#">. . . </a></li>
<li><a href="#">. . . </a></li>
```
Please kindly correct all other occurrences 🤗👍